Makefile

```diff
@@ -0,0 +1,3 @@
+lint:
+	@./node_modules/.bin/eslint .
```

Comment:
@a60814billy For further purpose, we should use Makefile to write test.
It make clearier to write differenct test in Makefile.
Refer: https://github.com/koajs/koa/blob/master/Makefile